layouts/partials/footer.hbs

```diff
@@ -15,13 +15,31 @@
         </ul>
       </div>
 
-      <p>© Node.js Foundation. All Rights Reserved. Portions of this site originally © Joyent.</p>
-      <p>Node.js is a trademark of Joyent, Inc. and is used with its permission. Please review the <a href="/static/documents/trademark-policy.pdf">Trademark Guidelines of the Node.js Foundation</a>.</p>
-      <p>Linux Foundation is a registered trademark of The Linux Foundation.</p>
-      <p>Linux is a registered <a href="http://www.linuxfoundation.org/programs/legal/trademark" title="Linux Mark Institute">trademark</a> of Linus Torvalds.</p>
-      <p>
-        <a href="https://raw.githubusercontent.com/nodejs/node/master/LICENSE">Node.js Project Licensing Information</a>.
-      </p>
+      <div class="linuxfoundation-content">
+        <div>
+          <p>© Node.js Foundation. All Rights Reserved. Portions of this site originally © Joyent.</p>
+          <p>Node.js is a trademark of Joyent, Inc. and is used with its permission. Please review the <a href="/static/documents/trademark-policy.pdf">Trademark Guidelines of the Node.js Foundation</a>.</p>
+          <p>Linux Foundation is a registered trademark of The Linux Foundation.</p>
+          <p>Linux is a registered <a href="http://www.linuxfoundation.org/programs/legal/trademark" title="Linux Mark Institute">trademark</a> of Linus Torvalds.</p>
+          <p>
+            <a href="https://raw.githubusercontent.com/nodejs/node/master/LICENSE">Node.js Project Licensing Information</a>.
+          </p> 
+        </div>
+
+        <div class="thanking-contributor">
+          <img id="contributor-avatar" loading="lazy" alt="Avatar of a Node.js contributor" src="/static/images/logos/js-green.svg">
+
+          <div>
+            <p>
+              Thanks <a href="#" rel="nofollow noopener noreferrer" target="_blank" rel="noreff" id="contributor-username">username</a> for being a <a href="">Node.js contributor</a>.
```

Comment:
BTW this should be `noreferrer` not `noreff`

Reply:
Nice catch, I forgot to remove the second rel